source/help/getting-started/organizing-conversations.rst

```diff
@@ -1,57 +1,79 @@
 Organizing Conversations
 ======================================
-_____
 
+Conversations in Mattermost are crucial to company productivity and success. Keeping information and messages organized in channels creates an efficient workplace.
+
+1. `Channel Types`_
+2. `Managing Channels`_
+
+-------------------------------------
 Channel Types
 -------------------------------------
-Channels are used to organize conversations across different topics. There are three types of channels: Public Channels, Private Channels, and Direct Messages.
+
+Channels are used to organize conversations across different topics. 
+There are three types of channels: Public Channels, Private Channels, and Direct Messages.
 
 Public Channels
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Public Channels are open to everyone on a team. New team members are automatically added to two Public Channels when they sign up: Town Square and Off-Topic.
 
 Private Channels
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Private Channels are for sensitive topics and are only visible to select team members. Any member of a Private Channel can add additional members. Channel members can choose to leave at any time, but only the channel owner or `Team Admin <http://docs.mattermost.com/help/getting-started/managing-members.html#user-roles>`_ can remove other members.
```

Comment:
@amyblais - would you be able to double check if this is correct for Team Edition as well? I think anyone may be able to remove people from channels in Team Edition, and in Enterprise Edition you can change the permissions to be Channel Admin and System Admin.

Reply:
@esethna Would you be able to help confirm that ^

Reply:
I believe LB is correct. Team Edition any channel members can add or remove members in public and private channels